webapp/src/main/webapp/resources/template.soy

```diff
@@ -552,3 +552,36 @@
   <p>Please delete the parent station of the station!</p>
   <br><br>
 {/template}
+
+/**
+ * Renders an explanation when route long and short name are equal.
+ * @param notices : List<[routeId : string, csvRowNumber : number, routeShortName : string, routeLongName : string]>
+ */
+{template .routeShortAndLongNameEqual}
+{let $numNotices: length($notices)/}
+  <p class="error">Error - Route Short and Long Name Equal!</p>
+  <p>Description: Both the short and long name of a route are equal.</p>
+  <p><b>{$numNotices}</b> found:</p>
+  <table>
+    <thead>
+      <tr>
+        <th>Route ID</th>
+        <th>CSV Row Number</th>
+        <th>Route Short Name</th>
+        <th>Route Long Name</th>
```

Comment:
If they are equal, why do we need two columns?

Reply:
True. I'll change it. Thanks

Reply:
Done!

Reply:
Can we update the screenshot?